chromium_src/net/socket/socks5_client_socket_auth.cc

```diff
@@ -69,7 +69,7 @@ int SOCKS5ClientSocketAuth::Authenticate(
       }
       case STATE_WRITE:
         DCHECK_EQ(OK, rv);
-        DCHECK_LT(0, buffer_left_);
```

Comment:
Please don't push this commit to this PR again, this also happens on master when (debug_build=true) and it is addressed in https://github.com/brave/muon/pull/584
I will drop this commit when I rebase to master again

Reply:
Got it.